ignite/pkg/cosmosgen/generate.go

```diff
@@ -13,17 +15,29 @@ import (
 	"github.com/ignite/cli/ignite/pkg/cosmosanalysis/module"
 	"github.com/ignite/cli/ignite/pkg/cosmosver"
 	"github.com/ignite/cli/ignite/pkg/gomodule"
+	"github.com/ignite/cli/ignite/pkg/xfilepath"
 )
 
 const (
-	moduleCacheNamespace = "generate.setup.module"
+	moduleCacheNamespace       = "generate.setup.module"
+	includeProtoCacheNamespace = "generator.includes.proto"
+)
+
+var protocGlobalInclude = xfilepath.List(
+	xfilepath.JoinFromHome(xfilepath.Path("local/include")),
+	xfilepath.JoinFromHome(xfilepath.Path(".local/include")),
 )
 
 type ModulesInPath struct {
-	Path    string
-	Modules []module.Module
+	Path           string
+	ModuleIncludes ModuleIncludes
 }
 
+var (
+	errFileFound    = errors.New("Searched file found")     // Hacky way to terminate dir walk early
+	errFileNotFound = errors.New("Searched file not found") // Hacky way to terminate dir walk early
+)
```

Comment:
Not sure about this specific case yet, but normally Go's standard is to write errors starting in lowercase, unless the first word is an abbreviation or proper noun:

```suggestion
var (
	errFileFound    = errors.New("searched file found")     // Hacky way to terminate dir walk early
	errFileNotFound = errors.New("searched file not found") // Hacky way to terminate dir walk early
)
```